public/assets/app.css

```diff
@@ -363,7 +363,7 @@ table a {
 }
 
 table tr td:first-child {
-  width: 100px;
+  width: 150px;
```

Comment:
With the font increase to `18px` when you're inside edit the number gets cut off at this width.